plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryTypeMapping.java

```diff
@@ -234,6 +234,43 @@ public void testNumericMapping()
                 .execute(getQueryRunner(), bigqueryCreateAndInsert("test.numeric"));
     }
 
+    @Test(dataProvider = "bigqueryNumericTypeProvider")
+    public void testNumericWriteMapping(String type, String value)
+    {
+        // insert operation performed through the Trino engine, not directly as in SqlDataTypeTest
```

Comment:
I don't understand this sentence. Could you explain why existing `trinoCreateAsSelect` or `trinoCreateAndInsert` doesn't work?

Reply:
Ok I think I need to rephrase it.
I want that test table to be created with column type NUMERIC, BIGNUMERIC in BigQuery (not through trino), but then I perform writes (INSERT), using trino.
And looks like we don't have such possibility in SqlDataTypeTest

Reply:
Please add `bigqueryCreateAndTrinoInsert` method then.

Reply:
refactored